plugins/catalog-backend/src/ingestion/processors/results.ts

```diff
@@ -53,7 +53,7 @@ export function generalError(
 
 export function location(
   newLocation: LocationSpec,
-  optional: boolean,
+  optional?: boolean,
```

Comment:
should mark this as deprecated too

Reply:
yep would probably work, think it's worth it?